tests/backup/backup_license_test.go

```diff
@@ -0,0 +1,140 @@
+			log.InfoD("Switching cluster context back to source cluster")
+			err = SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster")
+			err = LicenseCountCheck(ctx, licenseInspectRequestObject, int64(len(totalNumberOfWorkerNodes)-2))
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying license count after applying label portworx.io/nobackup=true to node %v", destinationClusterWorkerNodes[0].Name))
+		})
+		Step("Removing label portworx.io/nobackup=true from worker nodes and verifying the license count", func() {
+			log.InfoD("Removing label from worker node on source cluster on which label was applied earlier and verifying the license count")
+			err := Inst().S.RemoveLabelOnNode(sourceClusterWorkerNodes[0], "portworx.io/nobackup")
+			log.FailOnError(err, fmt.Sprintf("Failed to remove label portworx.io/nobackup=true from worker node %v", sourceClusterWorkerNodes[0].Name))
+			sourceClusterWorkerNodeLabelRemoved = true
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			err = LicenseCountCheck(ctx, licenseInspectRequestObject, int64(len(totalNumberOfWorkerNodes)-1))
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying license count after removing label portworx.io/nobackup=true from node %v", sourceClusterWorkerNodes[0].Name))
+			log.InfoD("Switching cluster context to destination cluster")
+			SetDestinationKubeConfig()
+			log.InfoD("Removing label from worker node on destination cluster on which label was applied earlier and verifying the license count")
+			err = Inst().S.RemoveLabelOnNode(destinationClusterWorkerNodes[0], "portworx.io/nobackup")
+			log.FailOnError(err, fmt.Sprintf("Failed to remove label portworx.io/nobackup=true from worker node %v", destinationClusterWorkerNodes[0].Name))
+			destinationClusterWorkerNodeLabelRemoved = true
+			log.InfoD("Switching cluster context back to source cluster")
+			err = SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster")
+			err = LicenseCountCheck(ctx, licenseInspectRequestObject, int64(len(totalNumberOfWorkerNodes)))
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying license count after removing label portworx.io/nobackup=true from node %v", destinationClusterWorkerNodes[0].Name))
+		})
+	})
+	JustAfterEach(func() {
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
+		if destinationClusterWorkerNodeLabelRemoved == false && destinationClusterWorkerNodeLabelApplied == true {
+			SetDestinationKubeConfig()
+			log.InfoD("Removing the applied label portworx.io/nobackup=true from worker nodes on destination cluster at the end of the testcase")
+			err = Inst().S.RemoveLabelOnNode(destinationClusterWorkerNodes[0], "portworx.io/nobackup")
+			dash.VerifySafely(err, nil, fmt.Sprintf("Failed to remove label portworx.io/nobackup=true from worker node %v", destinationClusterWorkerNodes[0].Name))
```

Comment:
`dash.VerifySafely` shows up as a green or red entry in Aetos. So, the message should not be an error message, rather it should be an assertion:
`fmt.Sprintf("Removing label portworx.io/nobackup=true from worker node %v", destinationClusterWorkerNodes[0].Name)`

Reply:
done